commons-shared/src/main/scala/com/avsystem/commons/collection/CollectionAliases.scala

```diff
@@ -12,105 +12,106 @@ import scala.{collection => sc}
   * `I` stands for immutable
   * `M` stands for mutable
   */
+//noinspection TypeAnnotation
```

Comment:
Please don't do this. Reconfigure IntelliJ and commit settings in `.idea`. Or just add these annotations.